app.rb

```diff
@@ -49,145 +74,176 @@ def create_person
   end
 
   def create_student
-    puts 'Enter Student Age: '
+    print "Enter the student's age: "
     age = gets.chomp.to_i
 
-    until age.positive?
-      puts 'Please Enter a valid age: '
-      age = gets.chomp.to_i
-    end
-
-    puts 'Enter Student Name: '
+    print "Enter the student's name: "
     name = gets.chomp
 
-    puts 'Does Student have parent permission to attend events? [Y/N]'
-    permission = gets.chomp.upcase
-
-    until %w[Y N].include?(permission)
-      puts 'Invalid Input. Please enter Y or N: '
-      permission = gets.chomp.upcase
-    end
+    print 'Has parent permission? [Y/N]: '
+    parent_permission_input = gets.chomp.downcase
+    parent_permission = %w[y yes].include?(parent_permission_input)
 
-    parent_permission = permission == 'Y'
-
-    student = Student.new(age, Classroom.new, name, parent_permission: parent_permission)
+    student = Student.new(name: name, age: age, parent_permission: parent_permission)
     @people << student
-    puts 'Student created successfully'
-  rescue StandardError => e
-    puts "An error occured: #{e.message}"
+
+    puts "Student created successfully! (ID: #{student.id})"
   end
 
   def create_teacher
-    puts 'Enter teacher name: '
-    name = gets.chomp
-
-    puts 'Enter teacher Age:'
+    print "Enter the teacher's age: "
     age = gets.chomp.to_i
-    until age.positive?
-      puts 'Please Enter a Valid Age: > 0'
-      age = gets.chomp.to_i
-    end
 
-    puts 'Enter field of specialization'
+    print "Enter the teacher's name: "
+    name = gets.chomp
+
+    print "Enter the teacher's specialization: "
     specialization = gets.chomp
 
-    teacher = Teacher.new(age, specialization, name)
+    teacher = Teacher.new(specialization: specialization, name: name, age: age)
     @people << teacher
 
```

Comment:
- [ ] @Team great work implementing the change requested, however, I am unable to access the already stored rental because the user id keeps changing after closing and opening the app as shown in the screenshot below, it would be nice if you stored the id and stop it from being randomly generated every time the user opens the app
![image](https://github.com/derrick1451/schools_library/assets/61408860/00e50f36-b97f-41f5-8ff4-0b0969dcbe7a)